addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -90,6 +90,37 @@ QUnit.test('composer text input: basic rendering', async function (assert) {
     );
 });
 
+QUnit.test('extended composer: basic rendering', async function (assert) {
+    assert.expect(2);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: 'channel',
+                id: 20,
+                mass_mailing: true,
```

Comment:
Not your fault but this name is so bad when we also have an actual mass mailing module.

Reply:
I would definitely change at least the commit title to "mailing channel" instead of "mass mailing"